doc/source/serve/deployment-graph/deployment_graph_intro.md

```diff
@@ -0,0 +1,219 @@
+(serve-intro-to-deployment-graphs)=
+
+# Intro to Deployment Graphs
+
+```{note}
+Note: This feature is in Alpha, so APIs are subject to change.
+```
+
+This section should help you:
+
+* compose your Ray Serve deployments together with the **Deployment Graph** API
+* serve your applications that use multi-model inference, ensemble models, ML model composition, or mixed business logic/model inference workloads
+* independently scale each of your ML models and business logic steps
```

Comment:
I didn't see this covered, maybe we just need a brief note about num_replicas somewhere?

Reply:
Good catch, I added a note.